libhb/hb_json.c

```diff
@@ -1091,6 +1103,7 @@ hb_job_t* hb_dict_to_job( hb_handle_t * h, hb_dict_t *dict )
                 "Forced",           unpack_b(&job->select_subtitle_config.force),
                 "Default",          unpack_b(&job->select_subtitle_config.default_track),
                 "Burn",             unpack_b(&subtitle_search_burn),
+                "ExternalFilename", unpack_s(&subtitle_search_external_filename),
```

Comment:
How would this work? What if there are multiple external subs?

Reply:
This is for *output* of external subtitle files, so you can do things like pair an external PGS file with mp4 output (since mp4 doesn't support embedded PGS).  Many players will import external subtitles.

The particular example you've highliighted is foreign audio search which will generate one output track.